app-modules/program-record-manager/src/Filament/Resources/ManageProgramResource.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+namespace AdvisingApp\ProgramRecordManager\Filament\Resources;
+
+use AdvisingApp\ProgramRecordManager\Filament\Resources\ManageProgramResource\Pages\CreateManageProgram;
+use AdvisingApp\ProgramRecordManager\Filament\Resources\ManageProgramResource\Pages\ListManagePrograms;
+use AdvisingApp\StudentDataModel\Models\Program;
+use App\Features\ManageStudentConfigurationFeature;
+use App\Filament\Clusters\ConstituentManagement;
+use App\Settings\ManageStudentConfigurationSettings;
+use Filament\Forms\Form;
+use Filament\Resources\Resource;
+use Filament\Forms\Components\DateTimePicker;
+use Filament\Forms\Components\TextInput;
+
+class ManageProgramResource extends Resource
+{
+    protected static ?string $model = Program::class;
+
+    protected static ?string $navigationIcon = 'heroicon-o-square-3-stack-3d';
+
+    protected static ?string $cluster = ConstituentManagement::class;
+
+    protected static ?string $navigationGroup = 'Students';
+
+    protected static ?string $label = 'Programs';
+
+    protected static ?int $navigationSort = 3;
+
+    public static function canAccess(): bool
+    {
+        /** @var User $user */
+        $user = auth()->user();
+
+        return ManageStudentConfigurationFeature::active() && $user->can('student_record_manager.configuration') && app(ManageStudentConfigurationSettings::class)->is_enabled;
```

Comment:
This should be checking the `view-any` permission you created. Not `configuration`